observations/observation_realtime_streaming.py

```diff
@@ -0,0 +1,1108 @@
+successLogger.addHandler(successBackuphandler)
+
+# Add the Errorloger
+errorLogger = logging.getLogger('error log')
+errorLogger.setLevel(logging.ERROR)
+errorBackuphandler = TimedRotatingFileHandler(f"{file_name_for_output_log}",when="w0",backupCount=1)
+errorLogger.addHandler(output_logHandler)
+errorLogger.addHandler(errorBackuphandler)
+
+# Add the Infologer
+infoLogger = logging.getLogger('info log')
+infoLogger.setLevel(logging.INFO)
+debug_logBackuphandler = TimedRotatingFileHandler(f"{file_name_for_debug_log}",when="w0",backupCount=1)
+infoLogger.addHandler(debug_logHandler)
+infoLogger.addHandler(debug_logBackuphandler)
+
+domArr = []
+
+kafka_url = config.get("KAFKA", "url")
+#consume the message from kafka topic
+app = faust.App(
+  'ml_observation_faust',
+  broker='kafka://'+kafka_url,
+  value_serializer='raw',
+  web_port=7001,
+  broker_max_poll_records=500
+)
+rawTopicName = app.topic(config.get("KAFKA", "observation_raw_topic"))
+producer = KafkaProducer(bootstrap_servers=[kafka_url])
+
+#db production
+# client = MongoClient(config.get('MONGO', 'url'))
+# db = client[config.get('MONGO', 'database_name')]
+# solCollec = db[config.get('MONGO', 'solutions_collection')]
+# obsCollec = db[config.get('MONGO', 'observations_collection')]
+# criteriaQuestionsCollec = db[config.get('MONGO', 'criteria_questions_collection')]
+# criteriaCollec = db[config.get('MONGO', 'criteria_collection')]
+# programsCollec = db[config.get('MONGO', 'programs_collection')]
+
+
+# # Define function to check if observation submission Id exists in Druid
+def check_observation_submission_id_existance(key,column_name,table_name):
+  try:
+      # Establish connection to Druid
+      url = config.get("DRUID","sql_url")
+      url = str(url)
+      parsed_url = urlparse(url)
+
+      host = parsed_url.hostname
+      port = int(parsed_url.port)
+      path = parsed_url.path
+      scheme = parsed_url.scheme
+
+      conn = connect(host=host, port=port, path=path, scheme=scheme)
+      cur = conn.cursor()
+      response = check_datasource_existence(table_name)
+      if response == True:
+          # Query to check existence of observation submission Id in Druid table
+          query = f"SELECT COUNT(*) FROM \"{table_name}\" WHERE \"{column_name}\" = '{key}'"
+          cur.execute(query)
+          result = cur.fetchone()
+          count = result[0]
+          infoLogger.info(f"Found {count} entires in {table_name}")
+          if count == 0:
+              return True
+          else:
+              return False
+      else:
+          # Since the table doesn't exist, return True to allow data insertion initially 
+          return True             
+  except Exception as e:
+      # Log any errors that occur during Druid query execution
+      errorLogger.error(f"Error checking observation_submission_id existence in Druid: {e}")
+   
+def check_datasource_existence(datasource_name):
+  host = config.get('DRUID', 'datasource_url')
```

Comment:
Handle this case. 
It might break the execution of the program if config is missed.